conans/client/rest/rest_client_v1.py

```diff
@@ -29,6 +30,8 @@ def complete_url(base_url, url):
 
 class RestV1Methods(RestCommonMethods):
 
+    threadpool = ThreadPool(8)
```

Comment:
Why not cpu count?

Reply:
I think it serves very different purpose to configure parallel builds, and it's actually a number of processes. here we configure thread pool, which is within the same process.